app/views/lead_providers/guidance/reference.html.erb

```diff
@@ -158,4 +158,104 @@
+
+<h3 id="csv-collection" class="govuk-heading-s">Create a new collection</h3>
+
+<p class="govuk-body">Create a New Collection and name it "CSV Started Declaration Submission"</p>
+
+<p class="govuk-body">For this collection, you will need to set <code>Authorization</code> to the following value:</p>
+<p class="govuk-body"><pre><code>
+Type: Bearer Token
+Token: {{bearerToken}}
+</code></pre></p>
+
+<p class="govuk-body">The <code>Pre-request script</code> should contain the following code to set the "request_body" previously set in the Body</p>
+<%=postman_code_sample(%Q[pm.environment.set('request_body', JSON.stringify({
+  "data": {
+    "type": "participant-declaration",
+    "attributes": {
+      "participant_id": data.id,
+      "declaration_date": data.declaration_date,
+      "declaration_type": "started",
+      "course_identifier": data.participant_type == "ect" ? "ecf-induction" : "ecf-mentor"
+    }
+  }
+}))])%>
+
+<p class="govuk-body">The <code>Test</code> should contain the following to determine the success of the declaration submission.</p>
+<%=postman_code_sample(%Q[pm.test("Send declaration via csv file", function(){
+  pm.response.to.have.status(200);
+})])%>
+
+<p class="govuk-body">
+  Save the collection using the icon at the top right
+</p>
+
+<p class="govuk-body">
+  This collection is a wrapper within which you will need a single test with an endpoint to run against. This test will contain just a single line which the pre-request script will update for each iteration of the test using the contents of a csv file chosen when running the test Using "Runner"
+</p>
+
+<h3 id="csv-collection" class="govuk-heading-s">Create a new request</h3>
+
+<p class="govuk-body">
+  Under the CSV Started Declaration Submission, select the "Add a request" to create the new action and endpoint.
+</p>
+
+<p class="govuk-body">
+  Name this test <code>Declare a participant from the CSV file</code>
+</p>
+
+<p class="govuk-body">
+  Set the operation type to <code>POST</code> and the endpoint to <code>{{baseUrl}}/api/v1/participant-declarations</code>
+</p>
+
+<p class="govuk-body">
+  Set the Body type to "raw" and enter
+  <code>
+    {{request_body}}
+  </code>
+</p>
+
+<p class="govuk-body">Save the request using the icon at the top right of the screen.</p>
+
+<h3 id="update-environment" class="govuk-heading-s">Populate the required environment</h3>
+
+<p class="govuk-body">Before making submissions, you need to create the two variables <code>{{baseUrl}}</code> and <code>{{bearerToken}}</code></p>
+
+<p class="govuk-body">These are required to be set as variables for the environment in which you wish to run this selection of declarations.</p>
+
+<p class="govuk-body">Choose the Variables tab and create the baseUrl and bearerToken variables.</p>
+
+<p class="govuk-body">Set these to the API URL address and Token you have for that location.</p>
```

Comment:
Providers are instructed to import the Swagger JSON file to import the collection. I think we would be better to document that here so that they don't need to set up each endpoint in the API. Did you see the video that I made that documents this?

Reply:
Ok. I haven't. I'll take a look. This is very specific for "started" participant-declarations and the scripts are more based on pulling data out of a supplied file. The body is just {{request_body}} and all of the heavy lifting is done via an integrated Pre-request script.